.github/workflows/ci.yml

```diff
@@ -105,20 +108,23 @@ jobs:
       - name: Set wait for image
         id: wait-for-image
         run: |
-          if [[ ${GITHUB_REGISTRY_WAIT_FOR_IMAGE} == "true" ]]; then
+          if [[ ${GITHUB_REGISTRY_WAIT_FOR_IMAGE} == 'true' ]]; then
               echo "::set-output name=wait-for-image::true"
           else
               echo "::set-output name=wait-for-image::false"
           fi
       - name: Selective tests
         id: selective-tests
-        run: ./scripts/ci/selective_tests.sh
+        run: ./scripts/ci/selective_tests.sh ${{github.sha}}
```

Comment:
Why github.sha here, and not the merge commit sha?

Reply:
The ``github.sha`` IS the merge sha for PRs (and for all other types of events we do not use it). It's not quite apparent, GH documentation is not clear about it entirely but you can read about it for example here: https://frontside.com/blog/2020-05-26-github-actions-pull_request/#how-does-pull_request-affect-actionscheckout

Reply:
Ah cool